src/App.jsx

```diff
@@ -1,5 +1,9 @@
+import { useState } from 'react';
```

Comment:
Improve imports order 

Useful link:
https://levelup.gitconnected.com/how-to-sort-imports-in-react-project-550f5ce70cbf